tools/poctools/makefile

```diff
@@ -18,7 +18,7 @@ build: .venv ## Build distribution files
 
 /dist/: build
 /build/: build
-publish: /dist/ /build/ ## Publish to PyPi
+publish: clean /dist/ /build/ ## Publish to PyPi
```

Comment:
Had to clean, or else old code would be uploaded to pypi.

Reply:
Nice one. I just kept doing it manually